ethcore/src/blockchain/blockchain.rs

```diff
@@ -689,7 +734,9 @@ impl BlockChain {
 		Some(TreeRoute {
 			blocks: from_branch,
 			ancestor: current_from,
-			index: index
+			index: index,
+			is_from_route_finalized: is_from_route_finalized,
+			is_to_route_finalized: is_to_route_finalized,
```

Comment:
still shouldn't have both of these

Reply:
(the only place where this does make sense is if `prospective.finalized = true`). But this information is already available to the caller.

Reply:
Yeah I think I got what you meant. If we use the finality definition settled down earlier, then `is_to_route_finalized` should always be `false`. I'm going to remove this variable.